layouts/partials/sections/featured-sessions.html

```diff
@@ -0,0 +1,115 @@
+                                            class="featured-sessions-section__avatar featured-sessions-section__avatar--first"
+                                        {{- else }}
+                                            class="featured-sessions-section__avatar"
+                                        {{- end }}>
+                                        {{- partial "shaped-image.html" (
+                                            dict
+                                            "imageResource" .avatarResource
+                                            "imageAlt" .name
+                                            )
+                                        }}
+                                        <figcaption class="featured-sessions-section__avatar-caption">
+                                            <strong>{{ .name }}</strong>
+                                            <div>{{ .slogan }}</div>
+                                        </figcaption>
+                                    </figure>
+                                {{- end }}
+                            {{- end }}
+
+                            {{- if gt ($session.speakers | len) 1 }}
+                                {{- with index $session.speakers 1 }}
+                                    <figure class="featured-sessions-section__avatar featured-sessions-section__avatar--second">
+                                        {{- partial "shaped-image.html" (
+                                            dict
+                                            "imageResource" .avatarResource
+                                            "imageAlt" .name
+                                            "hideDottedSquare" true
+                                            )
+                                        }}
+                                        <figcaption class="featured-sessions-section__avatar-caption">
+                                            <strong>{{ .name }}</strong>
+                                            <div>{{ .slogan }}</div>
+                                        </figcaption>
+                                    </figure>
+                                {{- end }}
+                            {{- end }}
+                        </div>
+                        <div class="featured-sessions-section__session">
+                            <h1 class="heading">
+                                {{ T "featured_sessions_section.heading" }}
+                            </h1>
+                            <h2 class="heading heading--2">
+                                {{ $session.sessionName }}
+                            </h2>
+                            <p class="paragraph paragraph--highlighted featured-sessions-section__session-description">
+                                {{ substr $session.sessionDescription 0 350 }} ...
+                                <a href="{{ $session.sessionUrl }}" class="link link--white">
```

Comment:
Both of these classes set the color, why do we need both?

Reply:
This is how BEM works, isn't it? You have a block (or element) and its modifiers. That both, in this case, have one and the same property is just accidentally and is likely to change over time.